src/mumble/API_v_1_x_x.cpp

```diff
@@ -1093,6 +1172,80 @@ void MumbleAPI::requestSetLocalUserComment_v_1_0_x(mumble_plugin_id_t callerID,
 	EXIT_WITH(MUMBLE_STATUS_OK);
 }
 
+void MumbleAPI::requestLinkChannels_v_1_3_x(mumble_plugin_id_t callerID, mumble_channelid_t *channelList,
+											std::size_t channelCount, std::shared_ptr< api_promise_t > promise) {
+	if (QThread::currentThread() != thread()) {
+		// Invoke in main thread
+		QMetaObject::invokeMethod(this, "requestLinkChannels_v_1_3_x", Qt::QueuedConnection,
+								  Q_ARG(mumble_plugin_id_t, callerID), Q_ARG(mumble_channelid_t *, channelList),
+								  Q_ARG(std::size_t, channelCount), Q_ARG(std::shared_ptr< api_promise_t >, promise));
+
+		return;
+	}
+	api_promise_t::lock_guard_t guard = promise->lock();
+	if (promise->isCancelled()) {
+		return;
+	}
+
+	VERIFY_PLUGIN_ID(callerID);
+
+	if (channelCount < 2) {
+		EXIT_WITH(MUMBLE_STATUS_OK);
+	}
+
+	const Channel *channel = Channel::get(static_cast< unsigned int >(channelList[0]));
+
+	if (!channel) {
+		EXIT_WITH(MUMBLE_EC_CHANNEL_NOT_FOUND);
+	}
+
+	for (size_t i = 1; i < channelCount; i++) {
+		const Channel *link = Channel::get(static_cast< unsigned int >(channelList[i]));
+		if (!link) {
+			EXIT_WITH(MUMBLE_EC_LINK_CHANNEL_NOT_FOUND);
+		}
+		Global::get().sh->addChannelLink(channel->iId, link->iId);
+	}
+
+	EXIT_WITH(MUMBLE_STATUS_OK);
+}
+
+void MumbleAPI::requestUnlinkChannels_v_1_3_x(mumble_plugin_id_t callerID, mumble_channelid_t channelID,
+											  mumble_channelid_t *unlinkList, std::size_t unlinkCount,
+											  std::shared_ptr< api_promise_t > promise) {
+	if (QThread::currentThread() != thread()) {
+		// Invoke in main thread
+		QMetaObject::invokeMethod(this, "requestUnlinkChannels_v_1_3_x", Qt::QueuedConnection,
+								  Q_ARG(mumble_plugin_id_t, callerID), Q_ARG(mumble_channelid_t, channelID),
+								  Q_ARG(mumble_channelid_t *, unlinkList), Q_ARG(std::size_t, unlinkCount),
+								  Q_ARG(std::shared_ptr< api_promise_t >, promise));
+
+		return;
+	}
+	api_promise_t::lock_guard_t guard = promise->lock();
+	if (promise->isCancelled()) {
+		return;
+	}
+
+	VERIFY_PLUGIN_ID(callerID);
+
+	const Channel *channel = Channel::get(static_cast< unsigned int >(channelID));
+
+	if (!channel) {
+		EXIT_WITH(MUMBLE_EC_CHANNEL_NOT_FOUND);
+	}
+
+	for (size_t i = 0; i < unlinkCount; i++) {
+		const Channel *unlink = Channel::get(static_cast< unsigned int >(unlinkList[i]));
+		if (!unlink) {
+			EXIT_WITH(MUMBLE_EC_LINK_CHANNEL_NOT_FOUND);
+		}
+		Global::get().sh->removeChannelLink(channel->iId, unlink->iId);
+	}
```

Comment:
The comments made on the link function also applies to the unlink function here.